template/catppuccin.user.css

```diff
@@ -47,19 +47,22 @@
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
 
+    /* Recommended base styles: */
+    color-scheme: if(@lookup = latte, light, dark);
+    accent-color: @accent-color;
```

Comment:
This is a debatable change here. Warrants further discussion before being merged. I believe this was briefly covered in the Discord forum but I can't remember who brought up the point that a red accent color could be confusing (potentially interpreted as a "dangerous" action?).